scripts/assign-colors

```diff
@@ -23,6 +23,16 @@ if __name__ == '__main__':
         metadata. If the metadata includes a 'focal' column that only contains
         boolean values, then restrict colors to traits for rows where 'focal'
         is set to True.""")
+    parser.add_argument('--ignore-categories', type=str, default=[], nargs='*',
+        help="""Do not create colors for these categories even if they are
+        included in the metadata and ordering TSV. This is useful for ignoring
+        categories in specific builds even if they share the same default
+        ordering TSV.""")
```

Comment:
[discussion] An alternative UI would be to have odering TSVs for different categories and then allow multiple `--ordering` values. Thoughts? Do we plan on vendoring some big ordering list, or copy-paste across repos?

Reply:
I suspect that there will be some overlap in ordering but not always. Even for division/location, the ordering may depend on where the pathogen is circulating? I'd think each pathogen would have it's own ordering TSV to support their own uses.